packages/@okta/vuepress-site/docs/guides/token-inline-hook/activate-and-enable/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Activate and enable
+---
+
+The Token Inline Hook must be activated and enabled within your Okta Admin Console.
+
+- Activating the Token Inline Hook registers the hook with the Okta org and associates it with your external service.
+- Enabling the Token Inline Hook associates the hook with your Okta custom authorization server, which authenticates the Okta-Hosted Login sample application.
+
+### Activate the Token Inline Hook
+
+1. Navigate to the **Workflow** > **Inline Hooks** page.
+
+2. Click **Add Inline Hook** and select **Token** from the drop-down menu.
+
+3. Add a name for the hook (in this example, "Patient Token Hook").
+
+4. Add your external service URL, including the endpoint. For example, use your Glitch project name with the endpoint: `https://your-glitch-projectname.glitch.me/tokenHook`.
+
+5. Include authentication field and secret. In this example:
+
+    - **Authentication field** = `authorization`
+    - **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+6. Click **Save**.
+
+The Token Inline Hook is now set up with a status of active.
+
+### Enable the Token Inline Hook
+
+1. Navigate to **Security** > **API** > **Authorization Servers**.
+
+2. Select a Custom Authorization Server from the list (usually **default**).
+
+3. Select the **Access Policies** tab and edit the policy rule to use with the hook. In most cases, edit the **Default Policy Rule**.
```

Comment:
There are actually 2 Edit icon on the Default Policy screen - the top Edit button isn't the one you want. It's the bottom pencil icon but there is no Edit label.  Can we make the instructions more clear on which edit to click on as I naturally gravitate to the top and looking for an "Edit" button and may not be aware that there is also Edit button at the bottom.. 
![Screen Shot 2021-02-26 at 4 35 12 PM](https://user-images.githubusercontent.com/67654866/109369390-06ff2280-7851-11eb-85d4-362c4d259cd9.png)

Reply:
Thanks, re-worded that step to orient the user towards the Rule table, i.e.: 

"Select the **Access Policies** tab. Navigate to the Rule table and click the Edit icon next to the policy rule that will use the Inline hook. In most cases, edit the **Default Policy Rule** of the **Default Policy**."